install/linux/usr/share/odemis/hwtest/orsay.odm.yaml

```diff
@@ -50,3 +54,18 @@ Orsay: {
     role: gis-reservoir,
     init: {}
 }
+
+"FIB-device": {
+    role: fib-device,
+    init: {}
```

Comment:
I think these init: {} are not required in the yaml. That said I think it's not in the way and maybe even looks better.